docs/native/src/testing/testcases.rst

```diff
@@ -1,3 +1,141 @@
+#. Click Back (<-)
+#. Click "Open"
+#. **RESULT: Confirm company service is shown as a place.**
+
+-----------------------------------
+2 - Open from Office - Fresh Install
+-----------------------------------
+*This test verifies the flow of using Company Service for the first time from Office.*
+
+#. Launch a fresh install of Office.
+#. Go through the First Run Experience.
+#. Skip Sign In.
+#. Go to Open -> Add a Place
+#. **RESULT: Company service shows up. Verify the name and icon of your service.**
+#. Select your Company Service.
+#. Enter credentials.
+#. **RESULT: Root folder should show.**
+#. Browse around the folder structure in your service.
+#. **RESULT: Browse works as expected.**
+#. Open a file from Browse.
+#. **RESULT: File should open in read-only mode.**
+#. Click sign in and sign in with a free Microsoft Account.
+#. **RESULT: File should open in edit mode if user is a consumer and read-only mode if the user is a commercial user.**
+#. Make changes [you will need to sign in with a subscription account for testing commercial user]
+#. Click Back (<-)
+#. Click "Open"
+#. **RESULT: File should have the previously saved changes. Ensure changes are being saved on Company service.**
+
+-------------------------------
+3 - Open from Company App - Repeat Usage
+-------------------------------
+Repeat test 1 except with company service already added (i.e. from previous usage).
+
+-------------------------------
+4 - Open from Office - Repeat Usage
+-------------------------------
+Repeat test 2 except with company service already added (i.e. from previous usage).
+
+-------------------------------
+5 - Save As [duplicate]
+-------------------------------
+Verify ability to duplicate to Company Service, both by adding a new place and using an existing place.
+
+|Duplicate|
+
+------------------------------
+6 - Create New [name]
+------------------------------
+Verify ability to duplicate to Company Service, both by adding a new place and using an existing place.
+
+------------------------------
+7 - Verify Licensing
+------------------------------
+Verify editing a file for a commercial user requires O365 subscription or else it opens read only.
+
+Please go to Settings -> [Microsoft App] -> Reset Word -> Delete Sign-In Credentials and restarting Office before doing this test.
+
+------------------------------
+8 - OAuth Login Page
+------------------------------
+Verify there is a link to the company's privacy statement on the company's login page when the user adds the company service as a place.
+
+Verify login page fits in window for various iPad and iPhone sizes.
+
+------------------------------
+9 - Verify File Properties 
+------------------------------
+Verify file properties from Recent and from opened file. When opening the properties from the Recent tab or the Open tab, the fields Author, Created, Modified By and Company will be empty.
+
+|ImageProps1|
+	*Properties view from Recent*
```

Comment:
Use `figure` instead of `image`, and you can provide a caption that will get automatically formatted properly. E.g.

```rst
    ..  figure:: /images/chrome_save_as_har.png
        :alt: An image showing the :guilabel:`Save as HAR with Content` option in the Chrome developer tools.

        :guilabel:`Save as HAR with Content` option in the Chrome developer tools
```